verified_boot_app.md

```diff
@@ -11,8 +38,108 @@ Src: https://github.com/Kicksecure/kicksecure-wiki-backup/blob/6900fdfbdea8693fd
 * Generic concept: yes
 * Architecture specific implementation: Yes, initially <code>amd64</code> only.
 * Architecture support: Initially <code>amd64</code> only. In principle, can be ported to other architectures by third parties.
-* Must be a motherboard deemed commercially profitable by the OEM: yes.
-* Must be a feature that the firmware developer considers potentially commercially beneficial (as in customers would buy motherboards with such freedom-preserving firmware): yes
+
+-->
+
+1.1. The application MUST be compatible with UEFI 2.7 Specification. Any
+     system compliant with the aforementioned specification version SHOULD be
+     capable of running this application.
+
+1.2. The application SHALL NOT support systems running in legacy
+     BIOS/compatibility mode (CSM).
+
+1.3. The application SHOULD be tested and verified on AMD64 architecture only.
+     However, the architecture support MAY NOT be limited to AMD64 only and
+     MAY also work on any architecture supported by the UEFI Specification 2.7
+     (refer to Section 2.3 Calling Conventions).
+
+<!--
+* Vendor neutral: yes, any operating system that wishes to be compatible with the "EFI_ALT" standard/convention is welcome to.
```

Comment:
wrap

Reply:
I left the original wiki content just for reference that the requirements cover all aspects mentioned in the wiki. And what the requirement map to.

They are subject to be removed.

Reply:
I'm not sure what to do with that. Will you remove it to resolve this discussion?

Reply:
Removed: 2e60f39